venvUtils/ensureVenv.py

```diff
@@ -21,6 +21,29 @@
 venv_python_version_path: str = os.path.join(venv_path, "python_version")
 
 
+def verifyExistingVenvPath():
+	"""
+	Verifies that the Python virtual environment at c{VENV_PATH} was actually created at that location,
+	and not just copied or moved there.
+	"""
+	# Activate the Python virtual environment and capture the content of the VIRTUAL_ENV environment variable.
+	existingPath = subprocess.check_output(
+		[
+			os.path.join(venv_path, "scripts", "activate.bat"),
+			"&&",
+			"call", "echo", "%VIRTUAL_ENV%",
+		],
+		shell=True,
+	).decode('oem').rstrip()

```

Comment:
Please could you add a comment here to show an example of the output for this command? If the output changes, this will make it easier to verify.